### PR TITLE
Add WhiteList info to deployment list

### PR DESCRIPTION
**Pull request by tarokkk**

### CODE REVIEW

File: pkg/helm/helm.go [hunk 1]
```diff
@@ -127,6 +127,7 @@ type ListDeploymentResponse struct {
 	Namespace    string `json:"namespace"`
 	CreatedAt    string `json:"createdAt,omitempty"`
 	Supported    bool   `json:"supported"`
+	WhiteListed  bool   `json:"whiteListed"`
```
[the remark this answers is not in the document]
Comment (tarokkk) [reply pkg/helm/helm.go]: We can give it another name but "securityScanWhiteListed" is too long in my opinion

[the remark this answers is not in the document]
Comment (bonifaido) [reply pkg/helm/helm.go]: If it is clearly documented in the OpenAPI descriptor I think we should keep it as short as possible.

File: api/helm.go [hunk 2]
```diff
@@ -153,6 +154,24 @@ func ListDeployments(c *gin.Context) {
 		}
 	}
 
+	// Create WhiteList set
+	securityClientSet := getSecurityClient(c)
+	if securityClientSet == nil {
+		log.Errorf("can't get security clientset: %s", err)
+	}
+	releaseWhitelist := make(map[string]bool)
+	if securityClientSet != nil {
```
Comment (bonifaido) [inline api/helm.go]: This should be an else if block I think.

Comment (tarokkk) [reply api/helm.go]: true & fixed

**Pull request merged**